src/sass/layouts/_header.scss

```
@@ -0,0 +1,114 @@
+header {
```

```suggestion
.header {
```
dùng class để style thay vì style trực tiếp cho element